kernels/border_align/border_align_backward/border_align_backward.cpp

```diff
@@ -79,30 +79,31 @@ mluOpStatus_t mluOpBorderAlignBackward(
   PARAM_CHECK(API, argmax_idx_desc->layout == MLUOP_LAYOUT_NHWC);
   PARAM_CHECK(API, grad_input_desc->layout == MLUOP_LAYOUT_NHWC);
 
-  PARAM_CHECK(API, grad_input_desc->dims[3] % border_num == 0);
-  PARAM_CHECK_NE(API, origin_n, 0);
-  PARAM_CHECK_NE(API, origin_c, 0);
-  PARAM_CHECK_NE(API, origin_h, 0);
-  PARAM_CHECK_NE(API, origin_w, 0);
-  PARAM_CHECK(API, origin_h * origin_w == origin_k);
+  PARAM_CHECK(API, grad_input_desc->dims[3] % 4 == 0);
```

Comment:
4 这种数字留个注释

Reply:
```suggestion
  PARAM_CHECK(API, grad_input_desc->dims[3] % border_num == 0, "(border_num = 4)");
```

这类防呆check，都可以加字符串，这样是不是好些？

Reply:
这种直接后面接常量字符串，PARAM_CHECK和PARAM_CHECK_V2 等价，可以这么写